extensions/google-shopping-ads/release-notes/index.md

```diff
@@ -16,6 +16,12 @@ The release notes include:
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 
+### v1.1.2
+
+Google Shopping ads Channel 1.1.2 is generally available for versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.fix} **Intermittent Load Error**: Resolved an intermittent error preventing Google Shopping ads Channel UI from loading after install.
```

Comment:
Is "UI" necessary? Can we remove it or is the distinction important?

Reply:
I didn't have UI in the draft.  it was added by product in review.   they wanted the distinction because the extension was working, but the screen wouldn't load in UI.   If there is a better distinction or a similar reference devdocs prefers, open to suggestion.

Reply:
Agree on that, could just remove UI from this line since it just isn't loading the extension at all? They don't receive any errors, timeout messages, etc? If so, might add that. If it just never loads, recommend the same as Jeff.

Reply:
If your reviewers requested it, then perhaps just spell it out.

Reply:
the extension does load and runs in the background....just the UI never loads on screen for the user.  discussed with ryan, and he agreed to drop "UI"